fabric-renderer-api-v1/src/main/java/net/fabricmc/fabric/api/renderer/v1/model/ModelHelper.java

```diff
@@ -56,6 +57,7 @@ public static int toFaceIndex(Direction face) {
 	 * optionally including the null face. (Use &lt; or  &lt;= {@link #NULL_FACE_ID}
 	 * to exclude or include the null value, respectively.)
 	 */
+	@Contract("null -> null")
```

Comment:
Is this really needed since the parameter is a primitive (int).

Reply:
If it was important enough to address in the JD, the anno is warranted.
Null int is 0, so maybe that is a better boundary to use? I don't know the details of the rendering API well enough to say.